cdk/resources/HealthCheckMqttBridge.ts

```diff
@@ -41,6 +43,27 @@ export class HealthCheckMqttBridge extends Construct {
 		})
 
 		// Lambda functions
+		const policies = nRFCloudAccounts.map(
+			(account) =>
+				new IAM.PolicyStatement({
+					actions: ['ssm:GetParametersByPath'],
+					resources: [
+						`arn:aws:ssm:${Stack.of(this).region}:${
+							Stack.of(this).account
+						}:parameter/${Stack.of(this).stackName}/thirdParty/${account}`,
```

Comment:
Simplify this by using wildcard.

```suggestion
						}:parameter/${Stack.of(this).stackName}/thirdParty/*`,
```